frontend/public/components/catalog/catalog-item-details.jsx

```diff
@@ -67,7 +70,18 @@ export class CatalogTileDetails extends React.Component {
             <div className="co-catalog-page__overlay-body">
               <PropertiesSidePanel>
                 {tileProvider && <PropertyItem label="Provider" value={tileProvider} />}
-                {supportUrl && <PropertyItem label="Support" value={supportUrlLink} />}
+                {(supportUrl || marketplaceSupportWorkflow) && (
+                  <PropertyItem
+                    label="Support"
+                    value={
+                      marketplaceSupportWorkflow ? (
+                        <ExternalLink href={marketplaceSupportWorkflow} text="Get Support" />
+                      ) : (
+                        supportUrlLink
```

Comment:
I'd think we'd want to render this as a link regardless of how it was set.

Reply:
It is; supportUrlLink is defined as a link further up. I moved marketplaceSupportWorkflow up there in the latest commit if that works.